src/tests/functional/plugin/shared/src/behavior/ov_infer_request/infer_request_dynamic.cpp

```diff
@@ -59,6 +60,16 @@ std::string OVInferRequestDynamicTests::getTestCaseName(testing::TestParamInfo<O
 void OVInferRequestDynamicTests::SetUp() {
     SKIP_IF_CURRENT_TEST_IS_DISABLED()
     std::tie(function, inOutShapes, targetDevice, configuration) = this->GetParam();
+    checkOutputValues = !strcmp(function->get_friendly_name().c_str(), "Sin");
```

Comment:
I guess better to add this flag into `OVInferRequestDynamicParams` instead comparison with function name

Reply:
As far as I remember we need to update instance for other plug-ins to pass `checkOutputValues` as test param. So let's do it.

Reply:
Also can we pass `std::function` that will execute reference? It looks more flexible. For `checkOutputValues == false` `std::function` should throw exception